Week2/home_work/Part1.js

```diff
@@ -0,0 +1,60 @@
+const mysql = require('mysql');
+const util = require('util');
+
+var connection = mysql.createConnection({
+   host: 'localhost',
+   user: 'hyfuser',
+   password: 'hyfpassword',
+   database: 'new_world'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function world() {
+
+   function countryCapital(x) {
+      const result = `SELECT country.name AS city, city.name AS capital FROM city, country WHERE id = capital AND country.name = ${x} `
+      return result;
+   }
+
+   function languagesInRegion(y) {
+      const result = `SELECT DISTINCT country.name, country.region FROM country JOIN countrylanguage on code = countrycode WHERE region = ${y} `
```

Comment:
You can reduce the amount of code by giving table names aliases => `countryLanguage as l`